### PR TITLE
explore: copy updates

### PR DESCRIPTION
**Pull request by pnavarrc**
- Remove "since March 1st" from the copy (From [Paper - CAN Explore](https://paper.dropbox.com/doc/CAN-Explore--A7ptCAakhRbthPiIlcrVsbKbAg-RIlOIJkMWHcX0AmXQA5PW))
- Change Trends → Raw Data [Slack thread](https://covidactnow.slack.com/archives/C018XQHEB1A/p1599778034007500)
- Change Hospitalizations and ICU Hospitalizations → COVID Hospitalizations and COVID ICU Hospitalizations (copy and tooltip). [Slack thread](https://covidactnow.slack.com/archives/C0110QQAPJA/p1600189192003700)

### CODE REVIEW

**Comment by pnavarrc** [comment]
[CAN Compare: Clarify that Hospitalization and ICU Hospitalization are for covid patients, not all patients.](https://trello.com/c/qwOetaEI/442-can-compare-clarify-that-hospitalization-and-icu-hospitalization-are-for-covid-patients-not-all-patients)

**Comment by pnavarrc** [comment]
[Copy updates for Explore](https://trello.com/c/SHcIHD0H/443-copy-updates-for-explore)

File: src/components/Explore/Explore.tsx [hunk 1]
```diff
@@ -98,10 +99,10 @@ const Explore: React.FunctionComponent<{
     <Styles.Container>
       <Grid container spacing={1}>
         <Grid item sm={6} xs={12}>
-          <Styles.Heading variant="h4">Trends</Styles.Heading>
+          <Styles.Heading variant="h4">Raw data</Styles.Heading>
```
Comment (mikelehen) [inline src/components/Explore/Explore.tsx:102]: Huh.  FWIW- I don't love this name. Seems very generic and I don't know if "raw" conveys anything useful.  Maybe "Data Explorer" or something?  I guess there was a discussion about this on slack already though, so if this is the consensus, okay.

Also, we call it the "trends" chart in the update text in the header at the top of the page.  But maybe that's fine. I don't know how long we'll keep that text anyway. 🤷

**Review by mikelehen — APPROVED** [review]

File: src/components/Explore/utils.ts [hunk 2]
```diff
@@ -141,35 +144,37 @@ export const exploreMetricData: {
   },
   [ExploreMetric.HOSPITALIZATIONS]: {
     title: 'Hospitalizations',
+    name: 'COVID Hospitalizations',
```
[the remark this answers is not in the document]
Comment (pnavarrc) [reply src/components/Explore/utils.ts:147]: ok, let's wait for the second/third wave of feedback 👍

File: src/components/Explore/Explore.tsx (hunk 1 above)
Comment (pnavarrc) [reply src/components/Explore/Explore.tsx:102]: ok, will revert to Trends then so it's consistent with the copy in the update.

**Comment by pnavarrc** [comment]
[CAN Compare: Clarify that Hospitalization and ICU Hospitalization are for covid patients, not all patients.](https://trello.com/c/qwOetaEI/442-can-compare-clarify-that-hospitalization-and-icu-hospitalization-are-for-covid-patients-not-all-patients)

**Pull request merged**